app/src/pages/inside/membersPage/membersGrid/projectRole/projectRole.jsx

```diff
@@ -0,0 +1,125 @@
+import { Fragment, PureComponent } from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames/bind';
+import { injectIntl, intlShape, defineMessages } from 'react-intl';
+import PropTypes from 'prop-types';
+import { fetch } from 'common/utils';
+import { URLS } from 'common/urls';
+import { InputDropdown } from 'components/inputs/inputDropdown';
+import { canChangeUserRole } from 'common/utils/permissions';
+import {
+  activeProjectSelector,
+  activeProjectRoleSelector,
+  accountRoleSelector,
+  isAdminSelector,
+} from 'controllers/user';
+import { ROLES_MAP, ADMINISTRATOR } from 'common/constants/projectRoles';
+import { showNotification } from 'controllers/notification';
+
+import styles from './projectRole.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  projectRoleMobile: { id: 'ProjectRole.mobileTitle', defaultMessage: 'Project role' },
+  allPermissions: {
+    id: 'ProjectRole.allPermissions',
+    defaultMessage: 'Administrator has full privileges according to permission map',
+  },
+  updateMember: {
+    id: 'ProjectRole.updateMember',
+    defaultMessage: 'Member {name} has been updated',
+  },
+});
+
+@injectIntl
+@connect(
+  (state) => ({
+    projectId: activeProjectSelector(state),
+    isAdmin: isAdminSelector(state),
+    canChangeRole: () =>
```

Comment:
This function will return the same result every time you call it (unless state changes).
Boolean flag makes more sense here:
`canChangeRole: canChangeUserRole(accountRoleSelector(state), activeProjectRoleSelector(state)),`